pkgs/development/python-modules/libpcap/default.nix

```diff
@@ -42,8 +43,15 @@ buildPythonPackage rec {
     pkg-about
   ];
 
-  # Project has tests, but I can't get them to run even outside of nix
-  doCheck = false;
+  preCheck = ''
+    pushd tests
+  '';
+  postCheck = ''
+    popd
+  '';
```

Comment:
```suggestion
  pytestFlagsArray = [ "tests" ];
```

Reply:
Unfortunately that does not work, due to how the tests are written. If you don't change the working directory, it cannot find the `pcaptestutils.py` file  :/